src/components/Link.js

```diff
@@ -13,24 +13,19 @@ class CSVLink extends React.Component {
   static defaultProps = commonDefaultProps;
   static propTypes = commonPropTypes;
 
+  static getDerivedStateFromProps(props) {
```

Comment:
Hey @djently , 
thanks for this change,
Could you please resolve conflicts.
Also I merged this PR #195 . Are they the same ? If they are the same, we can close this one. Otherwise, I will be waiting for your action.

Reply:
mine can be closed